src/SpatialBuilder.php

```diff
@@ -151,6 +151,19 @@ public function whereWithin(string $column, Geometry|string $geometryOrColumn):
     return $this;
   }
 
+  public function whereNotWithin(string $column, Geometry|string $geometryOrColumn): self
+  {
+    $this->whereRaw(
+      sprintf(
+        'ST_WITHIN(%s, %s)',
+        "`{$column}`",
+        $this->toExpression($geometryOrColumn),
+      ) . ' = 0 '
```

Comment:
This will make it a bit more readable
```suggestion
      sprintf(
        'ST_WITHIN(%s, %s) = 0',
        "`{$column}`",
        $this->toExpression($geometryOrColumn),
      )
```